src/client/components/GlobalError.tsx

```diff
@@ -75,7 +75,7 @@ export const GlobalError = ({ error, link, left }: GlobalErrorProps) => {
           <div>
             {error}
             &nbsp;
-            <ExternalLink href={link.link} css={errorLink} subdued={true}>
+            <ExternalLink href={link.link} css={errorLink} target="_blank">
```

Comment:
Amended the appearance of this as per gchat with Dom. Added `target="_blank"` for good measure as I don't think users will want to be navigated away just to report this issue?